mesop/web/src/app/styles.scss

```diff
@@ -260,6 +260,23 @@ body {
   }
 }
 
+// The box component has div-like semantics, so make it `display: block` by default.
+// We use a high-level selector first since this will allow custom CSS selectors
+// to override this style more easily, specifically for flex and grid displays.
+component-renderer {
+  display: block;
+}
+
+// Custom elements like Angular component tags are treated as inline by default.
+//
+// Since component-renderer encompasses many different types of a components, we
+// need to add a more specific selector to make these inline by default.
+component-renderer:not([mesop-box]),
+// The first component-renderer object is a box, but it needs to be inline.
```

Comment:
why does it need to be inline?

Reply:
That's a great question. I wasn't quite able to figure that out. But when I looked at the generated markup, that first component-renderer box element does not have display block by default on main branch (which is definitely strange since in component renderer ts code, in the main branch, it should set the style to display block, at least that's what I would have thought).

Reply:
Yeah I noticed this because some things definitely broke when I changed it to display block. I think the primary issue occurred in the chat apps where the full height stuff wasn't working. So seems like something that could be adjusted.